src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,14 +6,15 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-		<Version>1.2.0-preview</Version>
+		<Version>1.2.1-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</AssemblyName>
 		<PackageTags>Recoveryservices;SiteRecovery;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-        Updates in this release:
-			- Fixes corresponding to ARM compliance in swagger introducing new API version (2018-01-10) and taking other bugfixes like representing enums as strings.
-      ]]>
+		        Updates in this release:
+			- Introduced generate.ps1 & generated files out of latest published swagger json.
```

Comment:
This is repo-specific, package release notes are viewed on nuget and should only be pertinent to the SDK itself

Reply:
Corrected the release notes to have only SDK related changes.